app/models/redde/system_command.rb

```diff
@@ -0,0 +1,43 @@
+class Redde::SystemCommand
+  attr_accessor :action
+
+  ALLOWED_ACTIONS = %w(cache unicorn sidekiq reboot).freeze
+
+  def self.execute(action)
+    new(action).process
+  end
+
+  def initialize(action)
+    @action = action
+    Logger.new('log/commands.log').info "Received #{action} command"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.